frontend/.storybook/playFunctions.ts

```diff
@@ -319,3 +318,28 @@ export const fillConfirmationForm: PlayFunction<ReactRenderer> = async (
     },
   });
 };
+
+export const fillMarkListAsFinalForm: PlayFunction<ReactRenderer> = async (
```

Comment:
This can be in the stories file of the detail page, this files is ment to for generic reusable play functions.